smailyforprestashop.php

```diff
@@ -251,6 +259,27 @@ public function getContent()
                 $output .= $this->displayConfirmation($this->l('Abandoned cart settings updated'));
             }
         }
+        // RSS
+        if (Tools::isSubmit('smaily_submit_rss')) {
+            $category_id = pSQL(Tools::getValue('SMAILY_RSS_CATEGORY_ID'));
```

Comment:
Is there a reason why `pSQL` is used so much to escape values fetched from the database? Even if they are not used in queries or templates.

Reply:
Not sure what the reason is, but keeping up with consistency I used it as well.

Reply:
Removed